scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -1032,13 +1032,26 @@ export function AuthoringDirective(
                     }
                 }
 
-                $rootScope.$broadcast('item:nextStage', {
-                    stage: stageList[stageIndex],
-                    itemId: $scope.item._id,
-                    selectedStage: selectedStage,
-                    selectedDesk: selectedDesk,
-                    item: $scope.item,
-                });
+                if ($location.$$path === '/workspace') {
+                    api.save('move', {}, {
```

Comment:
We have a hook for to run before sending in extensions (https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/superdesk-api.d.ts#L107)

Is this hook relevant here, and will it still be fired?